rs/pocket_ic_server/src/pocket_ic.rs

```diff
@@ -163,6 +163,7 @@ pub struct PocketIc {
     registry_data_provider: Arc<ProtoRegistryDataProvider>,
     runtime: Arc<Runtime>,
     nonmainnet_features: bool,
+    log_level: Option<String>,
```

Comment:
Wouldn't it make more sense if this was type `Level`?

Reply:
Makes sense! Done.